### PR TITLE
Activity

### PR DESCRIPTION
**Pull request by d-opri**
add activity.js page, add List.js component, add db.json

### CODE REVIEW

**Comment by d-opri** [comment]
the styles for the component List.js will be changed at a later on, rows 27-113

**Review by Long-HaHoang — COMMENTED** [review]
You should be careful with figma's auto layout.
It often uses calc. You might get sub pixels

**Comment by Long-HaHoang** [comment]
I would suggest you to edit your db.json.
You are using the same id for multiple items.
In Line 10 of List.js, you map with those ids.
This will give you warnings.


I just cloned your project this is what I get:

Screenshot 2023-01-20 at 20 54 36

**Comment by d-opri** [comment]
> I would suggest you to edit your db.json. You are using the same id for multiple items. In Line 10 of List.js, you map with those ids. This will give you warnings.
> 
> I just cloned your project this is what I get:
> 


oh youre right! i had changed the ids at some point i remember but must have lost the file in my branches maybe.. thanks for the help!

File: components/List.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import React from "react";
+
+export default function List(props) {
+  return (
+    <>
+      {props.items.map((item) => (
+        <li key={item.id}>
```
Comment (RobinWitt) [inline components/List.js]: Hi Danie, in your preview I see you append the list items directly to the component.
The best practice for semantics would be to wrap your 'li''s into an 'ul' or 'ol'.

File: components/List.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+              {item.amount} $
+            </AmountText>
+          </ItemLayout>
+        </li>
+      ))}
+    </>
+  );
+}
+
+const ListLayout = styled.div`
```
Comment (RobinWitt) [inline components/List.js]: Here is a ListLayout, maybe you wanted to use this to style your list? Semanticly correct would be to use
` const ListLayout = styled.ul``
` 
or
`const ListLayout = styled.ol``
`
this you could use to wrap your list items into.

File: components/List.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  margin-left: 2em;
+  margin-right: 2em;
+`;
+
+const ItemLayout = styled.div`
```
Comment (RobinWitt) [inline components/List.js]: Same here, you could use 
`const ItemLayout = styled.li``
`
and drop the unnecessary div

**Review by RobinWitt — COMMENTED** [review]
I left you some comments regarding your List component file

Hope this feedback is helpful, I'm also open to feedback if I wasn't seeing something.

File: components/List.js (hunk 2 above)
Comment (d-opri) [reply components/List.js]: ah youre absolutely right, i missed that. thanks for the pointers!

**Review by RobinWitt — APPROVED** [review]

**Pull request merged**